### ISSUE TITLE
Breaking styles

### ISSUE THREAD

**Issue by red17electro** [stryberventures/stryber-react-ui-components-v2#154]
Hello everyone,

I am experiencing issues with styles breaking on button onClick / navigation change, etc. 
It happens mainly with buttons, and inputs.

Please check the attached gif:

![breakingstyles](https://user-images.githubusercontent.com/16454623/215064157-fa94a03e-a9ef-4d9d-933d-84db69464f44.gif)

version of components: latest.

```
    "@stryberventures/gaia-react.button": "^0.0.4",
    "@stryberventures/gaia-react.checkbox": "^0.0.4",
    "@stryberventures/gaia-react.chip": "^0.0.4",
    "@stryberventures/gaia-react.combobox": "^0.0.2",
    "@stryberventures/gaia-react.dropdown": "^0.0.5",
    "@stryberventures/gaia-react.form": "^0.0.3",
    "@stryberventures/gaia-react.input": "^0.0.5",
    "@stryberventures/gaia-react.input-password": "^0.0.4",
    "@stryberventures/gaia-react.multiselect": "^0.0.5",
    "@stryberventures/gaia-react.select": "^0.0.4",
    "@stryberventures/gaia-react.text": "^0.0.2",
    "@stryberventures/gaia-react.text-link": "^0.0.4",
    "@stryberventures/gaia-react.theme": "^0.0.3",
```

**Comment by SergeySychevskiy** [comment]
Hey Server. Thanks for sharing. We've seen this issue as well in case when parent component unmounts/mounts, but we didn't know that it can be reproduced during navigation change as well. Our team is working on this, we will let you know once we have any updates. Thanks!

**Comment by SergeySychevskiy** [comment]
The issue was fixed in 0.1.0 release :)
